modules/transactions.js

```diff
@@ -126,8 +126,10 @@ __private.list = function (filter, cb) {
 		filter.orderBy, {
 			sortFields: sql.sortFields,
 			fieldPrefix: function (sortField) {
-				if (['height', 'blockId', 'confirmations'].indexOf(sortField) > -1) {
+				if (['height'].indexOf(sortField) > -1) {
 					return 'b_' + sortField;
+				} else if (['confirmations'].indexOf(sortField) > -1) {
+					return sortField;
 				} else {
```

Comment:
So while adding tests I checked that queries with order by = blockId or confirmation returned an error. Looks like tests did the job...